modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+As a consequence, there were circumstances where cutting a list, or part of a list, to the clipboard did not behave as expected. In some circumstances, the remaining list was not the expected list, or the expected blank line did not present.
+
+With this update, the `list` plugin cleans up after cutting a {productname} list, or part thereof, as it does when a list, or part thereof, is deleted. Cutting to the clipboard now leaves the same result as deleting, as expected.
+
+
+=== Autocompleter was not triggered at the start of empty nested list items within non-empty list items
+
+The {productname} Autocompleter feature displays suggestions while the user is typing. If, however, the insertion point was at the start of an empty list item nested inside an outer list item that contained text, typing a trigger character (such as the `:`) did not produce the suggestions display, as expected. (If the outer list item was otherwise empty, the trigger character worked.)
+
+The underlying cause was Autocompleter using `dom.getRoot()` as its root element. As of this update, Autocompleter uses the nearest parent block element as its root element rather than `dom.getRoot()`. And, consequently, trigger characters cause the suggestions display to present in nested list items inside populated list items, as expected.
+
+
+=== Some template plugin option values were not escaped properly when doing replacement lookups with Regular Expressions
+
+The {productname} `template` plugin processes the following settings using regular expression (regex) matches:
+
+* `template_cdate_classes`
+
+* `template_mdate_classes`
+
+* `template_selected_content_classes`
+
+* `template_replace_values`
+
+* `template_preview_replace_values`
+
+Previously, however, it did not escape or correctly handle all the generated regex expressions.
+
+In some cases it did not escape settings values for safe use within a regex and, in one case, incorrect handling caused word boundaries — `+\b+` — to not work as expected.
+
+These errors have been corrected. In {productname} 6.1, template settings values are escaped properly. As well the incorrect handling has been corrected, and word boundaries work as expected.
+
+
+=== Copy events were not dispatched in readonly mode
+
+When a {productname} document was in read-only mode, copy events were captured and killed.
+
+This capture meant that the neither {productname}’s built-in copy-and-paste, nor the PowerPaste plugin’s copy-and-paste could intercept the copy event and act to copy (and, consequently, paste) the selected material.
+
+Instead, copying from a read-only {productname} document fell-back to the host browser’s copy command.
+
+In some circumstances, this resulted in the subsequently pasted material containing unexpected characters. Specifically, normal spaces were sometimes replaced with non-breaking spaces, `&nbsp;`.
```

Comment:
Code tags in adoc don't seem to be literal, they still allow HTML entities, so this needs to be double-encoded.

```suggestion
In some circumstances, this resulted in the subsequently pasted material containing unexpected characters. Specifically, normal spaces were sometimes replaced with non-breaking spaces, `&amp;nbsp;`.
```

Reply:
```suggestion
In some circumstances, this resulted in the subsequently pasted material containing unexpected characters. Specifically, normal spaces were sometimes replaced with non-breaking spaces, `+&nbsp;+`.
```
This is how to make it literal.